packages/swapper/src/swappers/CowSwapper/getCowSwapTradeQuote/getCowSwapTradeQuote.test.ts

```diff
@@ -326,14 +326,14 @@ const expectedTradeQuoteSmallAmountWethToFox: TradeQuote = {
   id: '123',
   receiveAddress: '0x0000000000000000000000000000000000000000',
   affiliateBps: '0',
-  rate: '14716.04718939437523468382', // 14716 FOX per WETH
+  rate: '144293027.59156357115',
```

Comment:
Looks odd but is absolutely correct, result of correct precision things now, see against prod:

<img width="1728" alt="Image" src="https://github.com/user-attachments/assets/86623015-54a6-4ee3-8800-e9495e04e7f4" />